packages/api-plugin-products/src/index.js

```diff
@@ -33,8 +33,8 @@ export default async function register(app) {
           [{ hashtags: 1 }, { name: "c2_hashtags" }],
           [{ shopId: 1 }, { name: "c2_shopId" }],
           [{ "workflow.status": 1 }, { name: "c2_workflow.status" }],
-          [{ createdAt: 1, shopId: 1 }, { name: "c2_createdAt_shopId" }],
-          [{ updatedAt: 1, shopId: 1 }, { name: "c2_updatedAt_shopId" }],
+          [{ createdAt: 1, shopId: 1, ancestors: 1 }, { name: "c2_createdAt_shopId" }],
+          [{ updatedAt: 1, shopId: 1, ancestors: 1 }, { name: "c2_updatedAt_shopId" }],
```

Comment:
As far as I know this will throw an error on start if you have ever started Reaction before as you cannot create a new index with the same name, you need to drop the index first?

Reply:
Ah interesting I assumed that it would true up - I confess I've spent several hours today dropping the db and re-creating it.